build-system-tests/scripts/mega-app-copy-files.sh

```diff
@@ -188,8 +193,8 @@ if [[ "$FRAMEWORK" == 'vue' ]]; then
         npx json -I -f mega-apps/${MEGA_APP_NAME}/tsconfig.json -e "this.allowJs=true"
     else
         echo "add allowJs: true to tsconfig for aws-exports.js"
-        echo "npx json -I -f mega-apps/${MEGA_APP_NAME}/tsconfig.json -e \"this.compilerOptions.allowJs=true\""
-        npx json -I -f mega-apps/${MEGA_APP_NAME}/tsconfig.json -e "this.compilerOptions.allowJs=true"
+        echo "npx json -I -f mega-apps/${MEGA_APP_NAME}/tsconfig.app.json -e \"this.compilerOptions.allowJs=true\""
+        npx json -I -f mega-apps/${MEGA_APP_NAME}/tsconfig.app.json -e "this.compilerOptions.allowJs=true"
```

Comment:
Missed this in the last PR where I updated the tsconfig to point to `tsconfig.app.json`, sorry about that!